test/e2e/specs/cache.js

```diff
@@ -58,7 +58,8 @@ describe('Backendless.Cache', function() {
       .then(() => expect(Cache.get('foo')).to.eventually.be.null)
   })
 
-  it('ttl', function() {
+  // TODO: this test gives floating results, need to be discovered
+/*  it('ttl', function() {
```

Comment:
for skipping a test you can use `x` before `it` or `describe` keywords

for example in your case should be `xit`